### ISSUE TITLE
Sample based on most Variable Genes

### ISSUE THREAD

**Issue by sgratzl** [Caleydo/caleydo#941]
for heatmap visualization instead of a random sampling an intelligent way would be interesting like the Most Variable Genes (~ the 1500 vom TCGA sample run)

**Comment by alexsb** [comment]
In heat maps there is no sampling currently, only in PCs. 

This essentially a issue of how to handle larger data-sets, which partially depends on #1216 

I'm not sure whether we should sample or simply provide a filtered stratification.

**Comment by sgratzl** [comment]
isn't this already done?

**Comment by mstreit** [comment]
realized in #1487
